uwhvz/settings/production.py

```diff
@@ -8,7 +8,7 @@
 ADMINS = [
```

Comment:
tbh this entire `Admins` block containing our personal emails shouldn't probably be existing as changes to a public repo 😬Debating whether we should actually cut this out entirely on Github, but leave in prod (for obvious reasons). @tso what do you think?